department-of-veterans-affairs/va.gov-cms
# VACMS-12709: Commit enable TMGMT and set permissions.

## Description

Closes #12709 and #12710

## Testing done
- Tests run
- Run export/import successfully

## Screenshots


## QA steps
What needs to be checked to prove this works?  

As user with the `translation_manager` role (may need to add the role to an existing user or create a new test user)
1. Visit sources page (https://pr13021-fn26fgyjzcrw4n3fiuz6upy4fonk0cla.ci.cms.va.gov/admin/tmgmt/sources). Filter to Support Services nodes (these are very short so good for testing). Check a few nodes, select English as the source and Spanish as the target. 'Request translation'. You will be taken to a new form. When you open up the Job items in the upper right, you see the various nodes you added in there.
   - [x] The items are visible in Job items
2. Provide a label for the job if you like. Click 'Submit to provider'. You will be taken back to the list of Sources. There will be an alert up top that says 'Exported file can be downloaded [here](http://va-gov-cms.ddev.site/sites/default/files/tmgmt_file/JobID5_en_es.xlf).' Confirm you can download the file. **Note:** you may get a download error due to security issues. You can either bypass the warning your browser gives you, or load the XLIFF file into a new tab and then save it to your machine. This will not be an issue in staging or prod.
   - [x] The file can be downloaded.
3. Download an XLIFF editor like Counterparts Lite. (https://michelf.ca/software/counterparts-lite/). Use this editor to enter test content for the file. Save it.
4. Go to the Jobs page ([/admin/tmgmt/jobs](https://pr13021-fn26fgyjzcrw4n3fiuz6upy4fonk0cla.ci.cms.va.gov/admin/tmgmt/jobs)). Find your job. Click 'manage'. Scroll to 'import file', add your 'translated' file, and import it.
- [x] The translation file is imported successfully.
5. Go back into 'manage' on your test job. You should see see your job items with yellow icons/colors labeling them, indicating they need to be reviewed. Review each item, and save as completed.
   - [x] Each item can be reviewed
   - [x] The translation matches what was entered in step 3.
   - [x] The translation can be saved as completed.
6. Go to the content admin (https://pr13021-fn26fgyjzcrw4n3fiuz6upy4fonk0cla.ci.cms.va.gov/admin/content). Filter to Support Service.
  - [x] You should see the nodes that you translated.
  - [x] If you visit or edit the translations, they should contain the translated content you provided in 3. 

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
